packages/micropip/test_micropip.py

```diff
@@ -61,24 +59,34 @@ def test_parse_wheel_url():
 
 def test_install_custom_url(selenium_standalone, web_server_tst_data):
     server_hostname, server_port, server_log = web_server_tst_data
-    selenium_standalone.load_package("micropip")
-    selenium_standalone.run("import micropip")
     base_url = f"http://{server_hostname}:{server_port}/"
     url = base_url + "snowballstemmer-2.0.0-py2.py3-none-any.whl"
-    selenium_standalone.run(f"micropip.install('{url}')")
-    # wait untill micropip is loaded
-    time.sleep(1)
-    selenium_standalone.run("import snowballstemmer")
+    selenium_standalone.run_js(
+        f"""
+        await pyodide.runPythonAsync(`
```

Comment:
We have `selenium_standalone.run_async` for this don't we?

Reply:
Yeah, but I prefer the tests to be styled so that they can be copied into the browser console when they fail. If you use `selenium_standalone.run_async` then when copying it you have to add an extra `pyodide.runPythonAsync()` around everything.

I've noticed that it's more common for errors to be generated in the `selenium.run` wrapper code than in the `selenium.run_js` wrapper code too (though gradually improving `run` is also probably a good thing), but again, such wrapper code errors are annoying because they are harder to reproduce interactively.

The bigger tradeoff is in consolidating calls to `selenium.run` or not consolidating them. If you consolidate the calls, then the selenium error message is less specific. I find that this is more than made up for when directly copying the test into console.html demonstrates the bug as well, because then I can easily see what step the error happened at and investigate the state just before and after.